CMakeLists.txt

```diff
@@ -167,7 +168,7 @@ if (Boost_FOUND)
 endif ()
 # We have to import LLVM's cmake definitions to build sea-dsa
 # Borrowed from sea-dsa's CMakeLists.txt
-find_package (LLVM 8.0.1 EXACT CONFIG)
+find_package (LLVM 9 CONFIG)
```

Comment:
Perhaps we could read this from `bin/versions` to eliminate redundancy here?

https://cmake.org/cmake/help/v3.0/command/file.html

Reply:
Done.